src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/missingswitchdefault/InputMissingSwitchDefaultCheckNullCaseLabel.java

```diff
@@ -0,0 +1,24 @@
+/*
+MissingSwitchDefault
+
+
+*/
+
+//non-compiled with javac: Compilable with Java21
+package com.puppycrawl.tools.checkstyle.checks.coding.missingswitchdefault;
+
+public class InputMissingSwitchDefaultCheckNullCaseLabel{
```

Comment:
We are still missing test cases. We need multiple case constants in the last label (like `null, default`), null case constant in a label preceding a default case constant, and switch expressions with the same. We also need to add cases with type patterns.

Reply:
Please check now @nrmancuso

Reply:
Still missing label with `null, default`.

Reply:
> Still missing label with `null, default`.

@nrmancuso Added label with null, default